### DISCUSSION TITLE
Accessibility Considerations

### DISCUSSION THREAD

**Discussion by commonroom** [shuding/nextra#288] [Ideas]
I would love to see a few accessibility features: 

* [skip to main link/bypass blocks (w3)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)
* `aria-expanded` attributes on navigation controls for [expandable content (IBM design system)](https://www.carbondesignsystem.com/components/accordion/code/)
* [accessible search combobox with listbox popup (w3)](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html)

This project looks so promising and I appreciate all the work so far.

**Comment by jermanuts** [comment]
Have you checked V2 docs in core branch?
